src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+.page {
+  margin: 0;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes`. While `.page` might be a class for the `body` element, if it's for any other tag, consider using a class instead of the tag name.